docs/cluster_spec.md

```diff
@@ -159,9 +158,39 @@ Will result in the flag `--feature-gates=ExperimentalCriticalPodAnnotation=true,
 
 ### networkID
 
-On AWS, this is the id of the VPC the cluster is created in. If creating a cluster from scratch, this field doesn't need to be specified at create time; `kops` will create a `VPC` for you.
+On AWS, this is the id of the VPC the cluster is created in. If creating a cluster from scratch, this field does not need to be specified at create time; `kops` will create a `VPC` for you.
 
 ```yaml
 spec:
   networkID: vpc-abcdefg1
 ```
+
+More information about re-using a VPC is [here](run_in_existing_vpc.md).
+
+### hooks
+
+Hooks allow a the execution of a container before the installation of Kubneretes on every node in a cluster.  For intance you can install nvidia drivers for using GPUs.
+
+```
+spec:
+  # many sections removed
+  hooks:
+  - execContainer:
+      image: kopeio/nvidia-bootstrap:1.6
+```
+
+Install Ceph
```

Comment:
Like: 
"How do I get RBAC working on Kops?"
"How do I install the dashboard?" ...  "Why is it broken with RBAC?" ;)
"How do I use Ceph with Kops?"
"How do I use CronJobs?"
"How do I use the Gossip instead of DNS for discovery?"

etc. etc.